src/components/v2v3/V2V3Project/ManageNftsSection/RedeemNftsModal/RedeemNftCard.tsx

```diff
@@ -0,0 +1,94 @@
+import axios from 'axios'
+import { IPFSNftRewardTier } from 'models/nftRewardTier'
+import { JB721DelegateToken } from 'models/subgraph-entities/v2/jb-721-delegate-tokens'
+import { MouseEventHandler } from 'react'
+import { useQuery, UseQueryResult } from 'react-query'
+import { twMerge } from 'tailwind-merge'
+import { classNames } from 'utils/classNames'
+import { cidFromIpfsUri, openIpfsUrl } from 'utils/ipfs'
+import { LoadingOutlined } from '@ant-design/icons'
+
+function useJB721DelegateTokenMetadata(
+  tokenUri: string | undefined,
+): UseQueryResult<IPFSNftRewardTier> {
+  return useQuery(
+    ['nft-rewards', tokenUri],
+    async (): Promise<IPFSNftRewardTier | undefined> => {
+      if (!tokenUri) return
+
+      const url = openIpfsUrl(cidFromIpfsUri(tokenUri))
+      const response = await axios.get(url)
+      const tierMetadata: IPFSNftRewardTier = response.data
+
+      return tierMetadata
+    },
+  )
+}
+
+export function RedeemNftCard({
```

Comment:
slight tech debt, a lot of duplicaiton from the normal NFT card. o well, needed for now

Reply:
Too easy